cpp/ql/src/experimental/Security/CWE/CWE-190/DangerousUseOfTransformationAfterOperation.ql

```diff
@@ -0,0 +1,94 @@
+/**
+ * @name Dangerous use of transformation after operation.
+ * @description By using the transformation after the operation, you are doing a pointless and dangerous action.
+ * @kind problem
+ * @id cpp/dangerous-use-of-transformation-after-operation.
```

Comment:
```suggestion
 * @id cpp/dangerous-use-of-transformation-after-operation
```
Query ids aren't supposed to have dots in them. That's also why the CI checks are failing currently.